src/pages/home/report/ReportActionsView.js

```diff
@@ -234,6 +234,16 @@ class ReportActionsView extends React.Component {
         if (didManuallyMarkReportAsUnread) {
             this.setState({newMarkerSequenceNumber: this.props.report.lastReadSequenceNumber + 1});
         }
+
+        // Preventing the subscribe event fail when the creating the report/workspace room optimistically. Check if the optimistic
+        // `OpenReport` or `AddWorkspaceRoom` success by `pendingFields.createChat` or `pendingFields.addWorkspaceRoom` is set to null.
+        // The other reports created will have empty field on `pendingFields`.
+        const didReportSuccessfullyCreated = !this.props.report.pendingFields
```

Comment:
Making this a bit cleaner
```suggestion
        const didCreateReportSuccessfully = !this.props.report.pendingFields
```